tests/test_util.py

```diff
@@ -15,3 +16,8 @@ def test_address_from_pubkey(self):
         pubkey = get_public_key_from_bytes_compressed(pubkey_bytes)
         address_b58 = get_address_b58_from_public_key(pubkey)
         self.assertEqual('HURjYEBdMPtk7kVYBKyHCWc3HAvjrx3unT', address_b58)
+
+
+class AsyncMock(MagicMock):
```

Comment:
Why not use `unittest.mock.Asyncmock`?

Reply:
It needs Python >= 3.8, right?

We need to discuss this with the team to increase the version required by this lib and tx-mining-service (there is already a positive initial feedback).

I'll create an issue for this.

Reply:
https://github.com/HathorNetwork/python-hathorlib/issues/15